client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicket.tsx

```diff
@@ -27,128 +47,187 @@ const Refund = () => {
+                type='email'
+                className='text-black border-2 border-gray rounded-l-full pl-12 w-80 py-2'
+                id='email'
+                placeholder='John_Smith@email.com'
+              />
+              <button id='email-search-input' type='submit' className='px-0'>
+                <div className='hover:bg-sky-500 w-20 rounded-r-full bg-indigo-500 hover:drop-shadow-md active:bg-sky-600'>
+                  <svg xmlns="http://www.w3.org/2000/svg" className="h-11 w-11 pl-6" viewBox="0 0 20 20" fill="white">
+                    <path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd" />
+                  </svg>
+                </div>
+              </button>
             </div>
-          </div>
-          <table className='w-full min-w-min'>
+          </form>
+          <table className={'w-full min-w-min'}>
             <thead>
               <tr className='grid grid-cols-5 gap-2 bg-gray-200 h-18 rounded-lg shadow-md px-2 mb-2 font-bold'>
                 <td className='row-start-1 justify-self-start py-2 col-span-1'>Name</td>
-                <td className='row-start-1 justify-self-start py-2 col-span-1'>Date</td>
-                <td className='row-start-1 justify-self-start py-2 col-span-1'>Showing(s)</td>
-                <td className='row-start-1 justify-self-start py-2 col-span-1'>Total</td>
+                <td className='row-start-1 justify-self-start py-2 col-span-1'>Order Date & Time</td>
+                <td className='row-start-1 justify-self-start py-2 col-span-1'>Event(s)</td>
+                <td className='row-start-1 justify-self-start py-2 col-span-1'>Order Total</td>
                 <td className='row-start-1 justify-self-center py-2 col-span-1'></td>
               </tr>
             </thead>
             <tbody>
-              {mappedInstances.length === 0 ? (
-                <tr className='text-center text-gray-600'>
-                  <td className='col-span-5'><p>No current results</p></td>
+              {orders.length === 0 ? (
+                <tr className="text-center text-gray-600">
+                  <td className={'col-span-5'}>
+                    <p>
+                    No Refundable Orders
+                    </p>
+                  </td>
                 </tr>
               ) : (
-                mappedInstances.map((instance, index) => (
-                  <tr key={index} className='grid grid-cols-5 gap-2 bg-gray-200 rounded-lg shadow-md px-2 mb-2 hover:bg-gray-300'>
-                    <td className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                orders.map((instance, index) => (
+                  <tr key={index} className = 'grid grid-cols-5 gap-2 bg-gray-200 rounded-lg shadow-md px-2 mb-2 hover:bg-gray-300'>
+                    <td className = 'row-start-1 justify-self-start pl-2 py-2 col-span-1'>
```

Comment:
Minor, but there shouldn't be spaces around the `=`

Reply:
fixed